Sources/PT.PM.JavaScriptParseTreeUst/JavaScriptEsprimaParseTreeConverter.cs

```diff
@@ -76,14 +78,17 @@ private Ust Visit(INode node)
             {
                 if (node is Statement statement)
                     return VisitStatement(statement);
-                else if (node is Expression expression)
+
+                if (node is Expression expression)
```

Comment:
What about converting these `if`s into single `switch` statement for clarity and consistency with other converters? What do you think?

Reply:
Good idea.